src/pages/networks/forms/NetworkFormMain.tsx

```diff
@@ -16,9 +16,10 @@ import { ensureEditMode } from "util/instanceEdit";
 interface Props {
   formik: FormikProps<NetworkFormValues>;
   project: string;
+  isClustered: boolean;
 }
 
-const NetworkFormMain: FC<Props> = ({ formik, project }) => {
+const NetworkFormMain: FC<Props> = ({ formik, project, isClustered }) => {
```

Comment:
maybe we can check if lxd is clusterd directly in this component instead of passing it down as prop?

Reply:
I did run into a "more hooks get rendered" error with the `useSettings` hook in the component and switching between network types. Hence, keeping the data loading in the parent to avoid it.

Reply:
I see, all good then